docs/dev/libs/cep.md

```diff
@@ -1324,7 +1324,7 @@ For example, for a given pattern `b+ c` and a data stream `b1 b2 b3 c`, the diff
 </table>
 
 Have a look also at another example to better see the difference between NO_SKIP and SKIP_TO_FIRST:
-Pattern: `(a | c) (b | c) c+.greedy d` and sequence: `a b c1 c2 c3 d` Then the results will be:
+Pattern: `(a | b | c) (b | c) c+.greedy d` and sequence: `a b c1 c2 c3 d` Then the results will be:
```

Comment:
I think `(a | b) (b | c) c+.greedy d` would be simpler and still valid, right?

Reply:
That was my first idea. But it seems that this pattern could not match c1 c2 c3 d too.
IMO, just skip the middle sequence while keeping the first and the last could help to understand.

Reply:
You're right. :( I'm afraid that if we remove the `(b | c)` it will not differ from `SKIP_TO_NEXT` which I would prefer to keep distinguished. Let's keep your suggestion from this PR.